### PR TITLE
Feature/basic category products

### PR DESCRIPTION
**Pull request by aneta4595**
- list of categories added
- detail simple card added
- products list by category showed
- add sorting menu-select but it's not working yet

### CODE REVIEW

File: src/app/components/category-products/category-products.component.html [hunk 1]
```diff
@@ -1 +1,176 @@
...
+          id="offcanvasCategory"
+          aria-labelledby="offcanvasCategoryLabel"
+        >
+          <div class="offcanvas-header d-lg-none">
+            <h5 class="offcanvas-title" id="offcanvasCategoryLabel">Filter</h5>
+            <!-- button -->
+            <button
+              type="button"
+              class="btn-close"
+              data-bs-dismiss="offcanvas"
```
Comment (gradzio) [inline src/app/components/category-products/category-products.component.html]: this is for the bootstrap js, we should avoid it actually. Please remove it

File: src/app/components/category-products/category-products.component.html [hunk 2]
```diff
@@ -1 +1,176 @@
...
+              <ul
+                class="nav nav-category"
+                id="categoryCollapseMenu"
+                *ngFor="let item of categoriesList$ | async"
+              >
+                <!-- w li zmiana zeby rozwijała się lista -->
+                <li
+                  class="nav-item border-bottom w-100 collapsed"
+                  data-bs-toggle="collapse"
+                  data-bs-target="#categoryFlushOne"
```
Comment (gradzio) [inline src/app/components/category-products/category-products.component.html]: same here, it is not best way. Try to look at ngx bootstrap way

File: src/app/components/category-products/category-products.component.html [hunk 3]
```diff
@@ -1 +1,176 @@
...
+                      }}</a>
+                    </h2>
+                    <div>
+                      <!-- rating -->
+
+                      <small class="text-warning">
+                        <i class="bi bi-star-fill"></i>
+                        <i class="bi bi-star-fill"></i>
+                        <i class="bi bi-star-fill"></i>
+                        <i class="bi bi-star-fill"></i>
```
Comment (gradzio) [inline src/app/components/category-products/category-products.component.html]: you are missing the real value here...

File: src/app/components/category-products/category-products.component.ts [hunk 4]
```diff
@@ -1,11 +1,63 @@
...
+
+
+  readonly oneCategory$: Observable<CategoryModel> =
+    this._activatedRoute.params.pipe(
+      switchMap((data) =>
+        this._categoriesService.getOneCategory(data['categoryId'])
+      )
+    );
+    readonly ordering$: Observable<{name:string}[]> = of([
+      { name: 'Featured' },
```
Comment (gradzio) [inline src/app/components/category-products/category-products.component.ts:47]: I think you are still missing some info here: which property do we sort? is it asc / desc ?

**Review by gradzio — CHANGES_REQUESTED** [review]

File: src/app/components/category-products/category-products.component.ts (hunk 4 above)
Comment (aneta4595) [reply src/app/components/category-products/category-products.component.ts:47]: probably yes :) I will do it next time, here I just started something

**Review by gradzio — APPROVED** [review]

**Pull request merged**